File: text/narwhal/read/text_read.js
#! /usr/local/bin/narwhal
// ------------------------------------------------------------------
//
//	text_read.js
//
//						May/07/2011
//
// ------------------------------------------------------------------
var text_manipulate = require ('/var/www/data_base/common/narwhal_common/text_manipulate.js');
// ------------------------------------------------------------------
print	("*** 開始 ***");
var args = require('system').args;
var file_in = args[1];

var dict_aa = text_manipulate.text_read_proc (file_in);
text_manipulate.dict_display_proc (dict_aa);

print	("*** 終了 ***");
// ------------------------------------------------------------------

File: text/octave/read/text_read.m
#! /usr/bin/octave -qf
#
#	text_read.m
#
#					Nov/11/2014
#
# ------------------------------------------------------------------
printf ("*** 開始 ***\n")
path (path, "/var/www/data_base/common/octave_common");
arg_list = argv ();
file_in = arg_list{1};
#
dict_aa = struct;
dict_aa = text_read_proc (file_in)
dict_display_proc (dict_aa)
printf ("*** 終了 ***\n")
# ------------------------------------------------------------------
